src/main/java/com/faforever/client/leaderboard/LeaderboardController.java

```diff
@@ -66,7 +68,13 @@ protected void onInitialize() {
 
     leagueSeasons.map(FXCollections::observableList).when(showing).subscribe(seasons -> {
       seasonPicker.getItems().setAll(seasons);
-      seasonPicker.getSelectionModel().selectFirst();
+      Optional<LeagueSeason> currentSeason = seasons.stream()
+                                                    .filter(season -> season.startDate().isBefore(OffsetDateTime.now()))
+                                                    .findFirst();
```

Comment:
Why are we filtering here instead of in the api call?

Reply:
Because I didn't think of doing it in the api call.
Theoretically this has the advantage that the future season is available in the picker, but I don't see any real advantage in that.